### PR TITLE
docs: Add a translation of README in Simplified Chinese

### PR DESCRIPTION
**Pull request by Octopus058**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,9 @@
 ##
 
 <div align="center">
-   <a href="/ENGLISH.md"> <b>Translate to english</b> </a>
+
+[English](README-en.md) | **한국어** | [简体中文](READNE-zhcn.md)
```
Comment (devxb) [inline README.md:7]: I think there is a typo. Please correct it.

```suggestion
[English](README-en.md) | **한국어** | [简体中文](README-zhcn.md)
```

**Review by devxb — CHANGES_REQUESTED** [review]
Thanks for contributions

File: README.md (hunk 1 above)
Comment (Octopus058) [reply README.md]: fixed

File: README-en.md [hunk 2]
```diff
@@ -3,7 +3,9 @@
 ##
 
 <div align="center">
-   <a href="/README.md"> <b>한국어로 번역하기</b> </a>
+
+**English** | [한국어](README.md) | [简体中文](READNE-zhcn.md)
```
Comment (devxb) [inline README-en.md:7]: I think there's a typo here too. 😭 

```suggestion
**English** | [한국어](README.md) | [简体中文](README-zhcn.md)
```

Comment (Octopus058) [reply README-en.md:7]: Ah well, maybe have to be tomorrow. I'm too tired to do any work, now is 12pm in my city. If any other typo u found, just comment here, Have a good night my friend :)

Comment (devxb) [reply README-en.md:7]: Of course, thank you so much for your interest and contribution, bro

**Review by devxb — APPROVED** [review]
Thanks for contributions @Octopus058 
Have a nice day 😄

**Comment by Octopus058** [comment]
u too my friend :D

**Pull request merged**